src/v2/Apps/Conversation/Routes/Conversation/index.tsx

```diff
@@ -26,14 +26,14 @@ interface ConversationRouteProps {
 
 const ConstrainedHeightFlex = styled(Flex)`
   height: calc(100vh - 145px);
-  ${media.xs`
-    height: calc(100vh - 55px);
+  ${media.md`
+    height: calc(100vh - 60px);
```

Comment:
Adjustment to address super tall section on small screens

Before:
<img width="878" alt="Screen Shot 2020-07-21 at 2 29 49 PM" src="https://user-images.githubusercontent.com/12748344/88092879-32a18800-cb5f-11ea-969a-b251da39439d.png">


<br>
<br>

After:
<img width="873" alt="Screen Shot 2020-07-21 at 2 29 25 PM" src="https://user-images.githubusercontent.com/12748344/88092890-36cda580-cb5f-11ea-950b-404b2f5efd76.png">